performance_tests/README.rst

```diff
@@ -0,0 +1,232 @@
+#####################################
+aws-encryption-sdk performance tests
+#####################################
+
+This module runs performance tests for the `AWS Encryption SDK Python`_.
+
+********
+Overview
+********
+
+This module tests the following keyrings / master key-providers:
+
+1. KMS Keyring / KMS Master Key Provider
+2. Raw AES Keyring / AES Master Key Provider
+3. Raw RSA Keyring / RSA Master Key Provider
+4. Hierarchy Keyring
+5. Caching CMM
```

Comment:
Do we know which MKP we plan to use with caching CMM?
I would suggest KMS MKP

Reply:
We're gonna use KMS MKP